backend/app/Http/Controllers/AuthController.php

```diff
@@ -69,4 +75,49 @@ public function update(UpdatePasswordRequest $request, User $user)
 
         return UserResource::make(['message' => 'Password updated successfully.']);
     }
+
+    public function forgotPassword(ForgotPasswordRequest $request)
+    {
+        // Send the password reset link
+        $status = Password::sendResetLink(
+            $request->only('email')
+        );
+
+        if ($status == Password::RESET_LINK_SENT) {
+            return response()->json(['message' => 'The link was sent, please check your email']);
+        } else {
+            return response()->json(['message' => trans('auth.email')], 401);
+        }
+
+        // Throw a validation exception if the password reset link sending failed
+        throw ValidationException::withMessages([
+            'email' => [trans($status)],
+        ]);
+    }
+
+    public function reset(ResetRequest $request)
```

Comment:
```suggestion
    public function reset?(ResetRequest $request)
```

be more descriptive for the function name